src/graphql/models/Article.js

```diff
@@ -529,6 +530,21 @@ const Article = new GraphQLObjectType({
       type: GraphQLString,
       description: 'Attachment hash to search or identify files',
     },
+    cooccurrences: {
+      type: new GraphQLList(Cooccurrence),
```

Comment:
nit: can add `GraphQLNonNull` on the list and the list item.

Reply:
ok